### ISSUE TITLE
NTR: Retinal edema (Eye structure & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7990]
New term request Retinal edema (elthat_150817162903):   UMLS: 0242420

**Comment by pnrobinson** [comment]
@Sukram is there a good definition for this? How should we delineate this from [Macular edema HP:0040049](https://hpo.jax.org/app/browse/term/HP:0040049)?

**Comment by pnrobinson** [comment]
@MickeySegal
can you provide the disease context? How can we differentiate this from Macular edema?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK534825/
"the orbital injury can lead to retinal edema"

https://ojrd.biomedcentral.com/articles/10.1186/1750-1172-7-20
" variety of eye lesions have been found including anterior uveitis, cataract, glaucoma, posterior segment involvement with vasculitis, vitritis, retinitis, panuveitis, retinal edema, cystoid macular degeneration, venous or arterial occlusion, disc edema and retinal detachment."

Also in Rosen's Emergency Medicine textbook, in Central retinal artery occlusion

So this is wider than just https://hpo.jax.org/app/browse/term/HP:0040049 or https://hpo.jax.org/app/browse/term/HP:0020120

**Comment by pnrobinson** [comment]
@MickeySegal 
This is the same as the existing term
[Intraretinal fluid HP:0031527](https://hpo.jax.org/browse/term/HP:0031527)
I am going to change the primary label of that term to Retinal edema and add the definition
def: Accumulation of fluid within the layers of the retina not limited to the macular region.
comment: The resulting swelling of the retina can lead to visual disturbances.
PMID: 29126927

**Comment by pnrobinson** [comment]
done
